src/opencl_keepass_fmt_plug.c

```diff
@@ -53,10 +53,13 @@ typedef struct {
 	uint32_t cracked;
 } result;
 
+#define AES_MAXNR   14
```

Comment:
FWIW, this addition of `AES_MAXNR` to `opencl_keepass_fmt_plug.c` looks unused.

Reply:
Good catch, that's a remnant from the older state struct.